yb-voyager/cmd/importDataFileCommand.go

```diff
@@ -153,12 +156,11 @@ func prepareCopyCommands() {
 }
 
 func setImportTableListFlag() {
-	tableList := []string{}
-	for _, tableName := range filePathToTableName {
-		tableList = append(tableList, tableName)
+	tableList := map[string]bool{}
```

Comment:
any specific reason to switch to `map` from `array`?

Reply:
There can be multiple `ImportFileTask` instances with the same target table name. We need unique names.